src/output-flex.cpp

```diff
@@ -1200,6 +1329,38 @@ output_flex_t::output_flex_t(std::shared_ptr<middle_query_t> const &mid,
             "No tables defined in Lua config. Nothing to do!"};
     }
 
+    // For backwards compatibility we add a "default" expire (with empty
+    // name) when the relevant command line options are used.
+    if (options.append && options.expire_tiles_zoom) {
+        auto &ts = m_tilesets->emplace_back("");
+        ts.set_filename(options.expire_tiles_filename);
+        ts.set_minzoom(options.expire_tiles_zoom_min);
+        ts.set_maxzoom(options.expire_tiles_zoom);
+
+        for (auto &table : *m_tables) {
+            if (table.has_geom_column() && table.geom_column().srid() == 3857) {
+                table.geom_column().add_expire({m_tilesets->size() - 1});
```

Comment:
Should that also take into account `get_options()->expire_tiles_max_bbox`?

Reply:
Ups. Yes.